src/build-scripts/gh-installdeps.bash

```diff
@@ -106,11 +110,12 @@ else
     fi
 
     if [[ "$USE_LIBHEIF" != "0" ]] ; then
-       sudo add-apt-repository ppa:strukturag/libde265 || true
-       sudo add-apt-repository ppa:strukturag/libheif || true
+       sudo add-apt-repository ppa:strukturag/libde265 -y
+       sudo add-apt-repository ppa:strukturag/libheif -y
+       time sudo apt-get update
        time sudo apt-get -q install -y libheif-plugin-aomdec \
             libheif-plugin-aomenc libheif-plugin-libde265 \
-            libheif-plugin-x265 libheif-dev || true
+            libheif-plugin-x265 libheif-dev -y
```

Comment:
The `|| true` was to keep CI from failing if ppa:strukturag was down, which I suspect must have been happening at some point.

I think there is room for debate about whether it's better to fail CI entirely if a repo is temporarily down, or whether it's fine to press on with the rest of the CI run but take the chance that if the repo stays down, maybe we wouldn't notice that we were not properly testing the heic support.

Reply:
Re-add ``|| true``